src/app/services/quickChart.service.ts

```diff
@@ -22,4 +26,15 @@ export class QuickchartService {
 
     return chartUrl;
   }
+
+  public getChartViaPost(chartData: ChartJSObject, format: string): Observable<unknown> {
+    const body = {
+      backgroundColor: '#fff',
+      width: 1000,
+      height: 500,
+      format: format,
+      chart: JSON.stringify(chartData),
+    };
+    return this.http.post('https://quickchart.io/chart/create', body);
```

Comment:
This will be the cause of the lack of custom branding.  Our QuickChart instance is at https://quickchart.micronutrient.support